net/yggdrasil/files/yggdrasil.sh

```diff
@@ -16,85 +16,69 @@ proto_yggdrasil_setup_peer_if_non_interface() {
 	local peer_config="$1"
 	local peer_address
 	local peer_interface
-	config_get peer_address "${peer_config}" "address"
-	config_get peer_interface "${peer_config}" "interface"
-	if [ -z ${peer_interface} ]; then
-		json_add_string "" ${peer_address}
-	fi;
+	config_get peer_address "$peer_config" "address"
+	config_get peer_interface "$peer_config" "interface"
+	[ -z "$peer_interface" ] && json_add_string "" "$peer_address"
 }
 
 proto_yggdrasil_dump_peer_interface() {
 	local peer_config="$1"
 	local peer_interface
 
-	config_get peer_interface "${peer_config}" "interface"
+	config_get peer_interface "$peer_config" "interface"
 
-	if [ ! -z ${peer_interface} ]; then
-		peer_interfaces="${peer_interfaces}\n${peer_interface}"
-	fi;
+	peer_interfaces="${peer_interfaces}${peer_interface:+\n$peer_interface}"
 }
 
 proto_yggdrasil_setup_peer_if_interface() {
 	local peer_config="$1"
 	local peer_address
 	local peer_interface
-	config_get peer_interface "${peer_config}" "interface"
-	if [ "${peer_interface}" = "${peer_interface_filter}" ]; then
-		config_get peer_address "${peer_config}" "address"
-		json_add_string "" ${peer_address}
+	config_get peer_interface "$peer_config" "interface"
+	if [ "$peer_interface" = "$peer_interface_filter" ]; then
+		config_get peer_address "$peer_config" "address"
+		json_add_string "" "$peer_address"
 	fi;
 }
 
 proto_yggdrasil_append_to_interface_regex() {
-	if [ -z "${regex}" ]; then
-		regex="$1"
-	else
-		regex="${regex}|$1";
-	fi;
+	regex="${regex:+$regex|}$1";
```

Comment:
hmm.. i try to avoid param expansion but this seems ok as first glance

the regex is for matching which interface(s) can rx/tx peering requests